samples/Tests/ColumnColumnSetContainer.Bleed.json

```diff
@@ -0,0 +1,315 @@
+{
```

Comment:
thanks for adding this test file.
can you break this into sevearl files?
It has several column sets and containers at top level.
since our parser doesn't tell us where it failed,
when parsing fails, it's really hard to debug it.

Reply:
Gilles told me to limit how many files we added to the tests as it would take more time to run the test app

---
In reply to: [264878238](https://github.com/Microsoft/AdaptiveCards/pull/2506#discussion_r264878238) [](ancestors = 264878238)